Jolt/Core/Core.h

```diff
@@ -191,6 +191,16 @@
 		#define JPH_USE_SSE4_1
 		#define JPH_USE_SSE4_2
 	#endif
+#elif defined(__ppc64__) || defined(_M_PPC64) || defined(__powerpc__) || defined(_M_PPC)
+//	#if defined(__ppc64__)
+	#define JPH_CPU_PPC64
+		#define JPH_CPU_ADDRESS_BITS 64
```

Comment:
We use JPH_CPU_ADDRESS_BITS to distinguish between 32 and 64 bit. So this should be JPH_CPU_PPC in both cases.